mlir/lib/ExecutionEngine/ROCm/BackendUtils.cpp

```diff
@@ -168,15 +161,6 @@ void BackendUtils::configTarget(std::string &targetChip,
     llvm_unreachable("HIP ArchName parsing should never fail.");
   }
 #else
-  std::string gcnArchName;
-  for (llvm::line_iterator lines(*gfxArchList); !lines.is_at_end(); ++lines) {
-    // Use the first Arch name found.
-    gcnArchName = lines->str();
-    break;
-  }
-  auto status = IsaNameParser::parseArchName(gcnArchName, targetChip, features);
-  if (status.failed()) {
-    llvm_unreachable("ArchName parsing failed.");
-  }
+  llvm_unreachable("BackendUtils.cpp does not include HIP.");
```

Comment:
Maybe more descriptive like: You are not supposed to use `configTarget` when compiling without HIP runtime.